configuration/monitor-acs.adoc

```diff
@@ -1,14 +1,46 @@
 [id="monitor-acs"]
-= Monitoring with Prometheus
+= Monitoring ACS
 include::modules/common-attributes.adoc[]
 :context: monitor-acs
 
 toc::[]
 
+= Monitoring with OpenShift 4
+
+[role="_abstract"]
+link:https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html[OpenShift Container Platform] includes a preconfigured,
+preinstalled, and self-updating monitoring stack that provides monitoring for core platform components.
+{product-title-short} has support for OpenShift Monitoring via encrypted and authenticated endpoint accessible only by Prometheus Operator.
+
+[IMPORTANT]
+====
+If you have previously configured monitoring with Prometheus Operator you may consider disabling it as it will be enabled by default on OpenShift 4.
+Having multiple ServiceMonitors could result in duplicating data and wrong query results.
```

Comment:
Shouldn't we rather advice to use the OpenShfit monitoring? I'm also not sure about the warning about wrong query results, it sounds a bit too scary for what it is imo.